access-mgmt/README.adoc

```diff
@@ -0,0 +1,469 @@
+* `Describe configs`
+
+|`Transactional ID`
+
+(For producer access to a resource)
+a|
+* `Starts with`
+* `Is`
+a|
+* `Allow`
+* `Deny`
+a|
+* `All`
+* `Write`
+* `Describe`
+
+|`Kafka instance`
+
+(For access to Kafka instance permissions in ACLs)
+|None
+a|
+* `Allow`
+* `Deny`
+a|
+* `Alter`
+* `Describe`
+|===
+
+[role="_additional-resources"]
+.Additional resources
+https://kafka.apache.org/documentation/#security_authz_primitives[Authorization Primitives^] in Kafka documentation
+
+[id="proc-setting-permissions_{context}"]
+== Setting account permissions in a Kafka instance in {product}
+
+[role="_abstract"]
+In {product}, you can create Access Control Lists (ACLs) in your Kafka instances and set permissions for how other user accounts or service accounts can interact with an instance and its resources. You can manage access for only the Kafka instances that you create or for the instances that the owner has enabled you to access and alter.
+
+.Prerequisites
+* You've created a Kafka instance and the instance is in *Ready* state.
+* The user account or service account that you're setting permissions for has been created in the organization.
+
+.Procedure
+. In the {product} web console, go to *Streams for Apache Kafka* > *Kafka Instances* and click the name of the Kafka instance that you want to set permissions for.
+. Click the *Permissions* tab to view the current ACL permissions for this instance.
++
+--
+By default, new Kafka instances contain the following permissions. These permissions allow all accounts in the organization to view the instance permissions and to view topics in the instance, but not to produce or consume messages.
+
+NOTE: The wildcard character `*` denotes any occurrences of the specified resource. For example, `Topic` = `is *` means any topic in a Kafka instance.
+
+.Default ACL permissions for new Kafka instances
+[cols="25%,25%,25%,25%"]
+|===
+h|Account
+h|Resource
+h|Access type
+h|Operation
+
+|`All accounts`
+|`Topic is *`
+
+(Any topic)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Consumer group is *`
+
+(Any consumer group)
+|`Allow`
+|`Describe`
+
+|`All accounts`
+|`Kafka instance`
+
+(Kafka instance permissions in ACLs)
+|`Allow`
+|`Describe`
+|===
+--
+. Use the *Permissions* page to set permissions for a new account, add permissions to an existing account, or delete account permissions in this instance.
++
+To set permissions for a new account in this instance, follow these steps:
+
+.. Click *Manage access*.
+.. In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for. You can also select *All accounts* to set permissions for all user accounts and service accounts in the organization.
+.. Click *Next*.
+.. Under *Review existing permissions*, verify the current permission settings for this instance. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Permissions* page.
```

Comment:
Maybe something like this instead?
> ...verify the current permission settings _for the selected account_ for this instance...

This section doesn't show all permissions for the instance, it only shows the permissions that affect the selected account.

Reply:
@jgiardino I'm not sure what you mean. As a user, I can _view_ all the permission settings regardless of what account I'm using, BUT I can only _update/delete_ other permission settings that are the same as what I'm adding, whether a single account or all accounts.

Can you clarify what you mean by "affect the selected account"? What permissions then wouldn't be shown?

Reply:
A few lines up, you have this step:
> In the *Account* drop-down menu, select the new user account or service account that you want to set permissions for.

After they complete this step, the permissions that display in the *Review existing permissions* are only the permissions that affect the account selected in that step. 

So if there are 3 ACLs defined for the Kafka instance, one for _All accounts_, one for _sterobin_, one for _jgiardino_, then this is what you'd see:
* If you selected _jgiardino_ in the *Account* dropdown, then you would see the ACLs for _All accounts_ and _jgiardino_, but not the one for _sterobin_
* If you selected _All accounts_ in the *Account* dropdown, then you would see the ACL for _All accounts_, but not the ACLs for _sterobin_ or  _jgiardino_

Reply:
Ah, I got it now. Sorry, I didn't realize that, but I see that now.

I've reworded then as follows. Again a tad wordy, but when I make it more concise, it's just not quite clear. This to me is clearest, though not as concise. Wdyt?

>The *Review existing permissions* section lists any permission settings for this instance that apply to all accounts in the organization and any permission settings that apply to the same account that you previously selected, if applicable. You can delete existing permissions now if needed, or you can wait to delete existing permissions later from the main *Access* page.

>If you previously selected a specific account, you can delete only permission entries that apply to individual accounts. If you previously selected *All accounts*, you can delete only permission entries that apply to all accounts.